services/src/main/java/io/grpc/services/ChannelzService.java

```diff
@@ -61,7 +61,7 @@ public static ChannelzService newInstance(int maxPageSize) {
     this.maxPageSize = maxPageSize;
   }
 
-  /** Returns top level channel aka {@link io.grpc.internal.ManagedChannelImpl}. */
+  /** Returns top level channel aka {@link io.grpc.ManagedChannel}. */
   @Override
   public void getTopChannels(
```

Comment:
An ordinary `//` comment here seems more suitable.
```
/** @inheritDoc */
// Returns top level channel aka {@link io.grpc.internal.ManagedChannelImpl}.
```

After all, the javadoc of its super class is available.

Reply:
I get the error below. It think it may be because the protobuf generated javadoc are inside `<pre>` tags, so the linter does not treat it as a sentence:
` First sentence of Javadoc is incomplete (period is missing) or not present. [SummaryJavadoc]`

Reply:
Hmm, interesting.